core/server/logging/PrettyStream.js

```diff
@@ -0,0 +1,144 @@
+// jscs:disable
```

Comment:
Wasn't lot of work, maybe 30mins. I just wrote the way we log, but not how pretty streaming works. This is a general approach and copied. 

There was no satisfying stdout module.